executor/common_fuchsia.h

```diff
@@ -118,7 +122,7 @@ static void install_segv_handler()
 #define NONFAILING(...)                                              \
 	{                                                            \
 		__atomic_fetch_add(&skip_segv, 1, __ATOMIC_SEQ_CST); \
-		if (_setjmp(segv_env) == 0) {                        \
+		if (setjmp(segv_env) == 0) {                        \
```

Comment:
Is there sigsetjmp? Ideally we want sigsetjmp(segv_env, 0). We don't need to save/restore signals, and that's a system call per argument copy. Or any of this is not relevant to fuchsia?

Reply:
We do have sigetsetjmp! Updated accordingly.